dali/test/argument_key.h

```diff
@@ -15,27 +15,50 @@
 #ifndef DALI_TEST_ARGUMENT_KEY_H_
 #define DALI_TEST_ARGUMENT_KEY_H_
 
-#include <utility>
 #include <string>
+#include <tuple>
+#include <assert.h>
 
 
 namespace dali {
 namespace testing {
 
-class ArgumentKey : public std::pair<std::string, std::string> {
+class ArgumentKey {
  public:
-  using Base = std::pair<std::string, std::string>;
+
+  ArgumentKey(const char *arg_name) noexcept : // NOLINT (non-explicit ctor)
```

Comment:
Style: when initializer lists is moved to a new line, move the colon, too.

Reply:
Done